test/cpp/jit/test_gpu.cpp

```diff
@@ -3066,80 +3066,188 @@ void testGPU_FusionSimpleGemm() {
   }
 }
 
-// This test currently requires a combination of broadcast and reduction
-// operations and parellelization strategy that is currently not supported.
-// It is a goal to get this example working and this test is added so we
-// can continue working on getting this example fixed. Right now it
-// produces an incorrect result. Either we need to error coherently on the
-// optimization strategy we don't support and set this test to one we do support
-// or we need to get this schedule working correctly.
+// Softmax without input normalization. This is a simplified version
+// of testGPU_FusionSoftmaxNormalized.
 void testGPU_FusionSoftmax() {
   torch::jit::fuser::cuda::CudaKernel prog;
   Fusion& fusion = *prog.fusion_;
   FusionGuard fg(&fusion);
 
+  const int tidx = 32;
+  const int dimx = 32;
+  const int dimy = 16;
+  const int dimz = 130;
+
   // Set up your input tensor views
   TensorView* input_tv0 = makeDummyTensor(3);
   fusion.addInput(input_tv0);
 
+  TensorView* exp_tv4 = unaryOp(UnaryOpType::Exp, input_tv0);
+  TensorView* sum_exp_tv5 =
+      reductionOp(BinaryOpType::Add, {2}, new Float(0), exp_tv4);
```

Comment:
why not using sum() ?

Reply:
Nothing particular. It was just written that way previously.

Reply:
makes sense. 

would you mind changing it to use sum()? not only easier to read, but these tests are also serving as examples for future developers and we want them to emulate using the recommended APIs.

Reply:
Sure. We may also want to extend arith.h/arith.cpp since we are missing some shortcuts. For example, we don't have `max`.